### PR TITLE
fix(Language): update language static data

### PR DESCRIPTION
**Pull request by anta-semenov**
Need for https://github.com/languagedrops/drops-frontend/issues/8052

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "rm -rf dist && tsc",
+    "build": "yarn test && rm -rf dist && tsc",
```
Comment (fadani) [inline package.json:8]: how come that we would need to run the tests every time we want to transpile the TS files?

Comment (anta-semenov) [reply package.json:8]: Yes, probably it's an overkill. The thing is that we don't have any auto-deployment flow for intersection and my worry here is that we can just forget that we need to run tests

Comment (anta-semenov) [reply package.json]: fixed

**Comment by anta-semenov** [comment]
I removed tests from build script. And will merge this for now. Feel free to leave comments

**Comment by fadani** [comment]
👍

**Pull request merged**